var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -53,11 +56,36 @@ def install(self, spec, prefix):
                 'TBB=-L%s -ltbb' % spec['tbb'].prefix.lib,
             ])
 
+        # --------------- Locate BLAS and LAPACK
+        exts = ['.a', '.so']
+        if ('+fpic' in spec):
+            exts.reverse()
+
+        # Find BLAS for the not-so-powerful makefile
```

Comment:
should not you use the interface introduced here https://github.com/LLNL/spack/pull/657
that is `self.spec.blas_shared_lib` and alike?